src/actor/register.rs

```diff
@@ -127,6 +127,23 @@ where
     type State = RegisterActorState<ServerActor::State, u64>;
     type Timer = ServerActor::Timer;
 
+    fn name(&self) -> String {
+        match self {
+            RegisterActor::Client {
+                put_count: _,
+                server_count: _,
+            } => "Client".to_owned(),
```

Comment:
Reminder for fix-up later: I think `{ .. }` syntax would suffice.

Reply:
Done in https://github.com/stateright/stateright/commit/1466b3ca650a53c6910107ae95500e9664c8ad50